package.json

```diff
@@ -16,15 +16,18 @@
     "fifo": "^2.3.0",
     "mic": "^2.1.1",
     "object.pick": "^1.2.0",
-    "pigpio": "^0.4.0",
-    "rpi-ws281x-native": "^0.8.1",
     "semaphore": "^1.0.5",
     "sleep": "^5.0.0",
     "temp": "^0.8.3",
     "twitter": "^1.5.0",
     "web-audio-api": "^0.2.2",
     "watson-developer-cloud": "^2.11.1"
   },
+  "optionalDependencies": {
+    "//": "pigpio and pi-ws281x-native are raspberry exclusive. To the project works in Linux it should be keeped as optional.",
```

Comment:
@victordibia do we have a compelling argument to maintain Linux compatibility outside of RPi?  it might make it easier to develop, but i feel it goes against the spirit of the project -- this lib is intended only for use on RPi.